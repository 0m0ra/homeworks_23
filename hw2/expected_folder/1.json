{
    "min age": 35,
    "max age": 43,
    "avg age": 38,
    "median age": 38,
    "< 2 days": 0,
    "< 1 week": 0,
    "< 1 month": 0,
    "< 6 months": 33.3,
    "> 6 months": 66.7
}