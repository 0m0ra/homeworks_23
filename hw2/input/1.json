{
    "user": {
        "region": "Kirov",
        "registered": "2019-06-06",
        "last_login": "2023-11-07",
        "email": "user@mail.ru",
        "age": 17
    },
    "user2": {
        "region": "Moscow",
        "registered": "2012-08-08",
        "last_login": "2022-05-21",
        "email": "user2@yandex.com",
        "age": 33
    }
}