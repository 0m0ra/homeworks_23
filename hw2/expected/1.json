{
   "0-18": 50.0,
   "18-25": 0.0,
   "26-45": 50.0,
   "45-59": 0.0,
   "60+": 0.0,
   "2019": 50.0,
   "2012": 50.0
}